whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticSearchInstance.java

```diff
@@ -0,0 +1,54 @@
+package net.ripe.db.whois.api.elasticsearch;
+
+
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpHost;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+@Profile({WhoisProfile.DEPLOYED})
+@Primary
+@Component
+public class ElasticSearchInstance implements ElasticRestHighlevelClient {
+    private final RestHighLevelClient client;
+
```

Comment:
Should we add the logger here as well?
private static final Logger LOGGER = getLogger(ElasticIndexService.class);

Reply:
Do we need to log something here?

Reply:
Actually I should catch all exception here and log as warning.